src/core/constraint.jl

```diff
@@ -1,21 +1,21 @@
 "model current constraints"
-function constraint_tp_model_current(pm::_PMs.GenericPowerModel; kwargs...)
+function constraint_mc_model_current(pm::_PMs.AbstractPowerModel; kwargs...)
     for c in _PMs.conductor_ids(pm)
         _PMs.constraint_model_current(pm; cnd=c, kwargs...)
     end
 end
 
 
 "reference angle constraints"
-function constraint_tp_theta_ref(pm::_PMs.GenericPowerModel, i::Int; nw::Int=pm.cnw)
+function constraint_mc_theta_ref(pm::_PMs.AbstractPowerModel, i::Int; nw::Int=pm.cnw)
     for cnd in _PMs.conductor_ids(pm)
-        constraint_tp_theta_ref(pm, nw, cnd, i)
+        constraint_mc_theta_ref(pm, nw, cnd, i)
     end
 end
 
 
 "storage loss constraints"
-function constraint_tp_storage_loss(pm::_PMs.GenericPowerModel, n::Int, i, bus, r, x, standby_loss)
+function constraint_mc_storage_loss(pm::_PMs.AbstractPowerModel, n::Int, i, bus, r, x, standby_loss)
```

Comment:
Based on changes in the constraint template to use the function defined in PowerModels, this function should not be called and can be removed.

Reply:
Good catch! its definitely not called, or there's be an error, but I missed it when I was pruning functions.